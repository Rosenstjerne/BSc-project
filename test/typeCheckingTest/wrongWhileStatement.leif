while (5) print(5);