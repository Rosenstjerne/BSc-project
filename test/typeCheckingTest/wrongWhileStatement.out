
Error in phase Type Checking, line 2:
    An expression of type 'int' cannot be used as a guarde. Must be of type 'bool'
